journal.py

```diff
@@ -76,77 +107,35 @@ def read_entry(request):
 def edit_entry_view(request):
     """return a list of all entries as dicts"""
     if request.authenticated_userid:
+        entry = Entry.by_id(request.params.get('id', None))
         if request.method == 'GET':
-            # import pdb; pdb.set_trace();
-            cursor = request.db.cursor()
-            cursor.execute(DB_ENTRY, (request.params.get('id', None), ))
-            keys = ('id', 'title', 'text', 'created')
-            row = cursor.fetchone()
-            entry = dict(zip(keys, row))
-            entry['created'] = entry['created'].strftime('%b %d, %Y')
-
-            return entry
+            return entry.json_edit_get()
 
         elif request.method == 'POST':
-            # import pdb; pdb.set_trace()
             try:
-                edit_entry(request)
+                entry.update_from_request(request)
             except psycopg2.Error:
```

Comment:
now that you're using SQLAlchemy, the errors you might get from running `update_from_request` will not be psycopg2 errors.  You should update this, or remove it.